src/Entities/Entity.h

```diff
@@ -530,6 +535,11 @@ class cEntity
 	eEntityType m_EntityType;
 	
 	cWorld * m_World;
+
+	/// State variables for ScheduleMoveToWorld.
+	bool m_IsWorldChangeScheduled = false;
```

Comment:
Yeah, sorry for the comment style, we couldn't decide on the format beforehands, so one half of the code uses `///` and the other half uses `/** */`. We've since then decided to use the latter, but didn't fix the former yet. Good call about replacing them all.